selendroid-standalone/src/main/java/io/selendroid/standalone/server/model/SelendroidStandaloneDriver.java

```diff
@@ -164,8 +164,12 @@ public void addToAppsStore(File file) throws AndroidSdkException {
         // using "android" as the app name, because that is the desired capability default in
         // selenium for
         // DesiredCapabilities.ANDROID
+          File androidAPK = androidDriverAPKBuilder.extractAndroidDriverAPK();
+          if(serverConfiguration != null && serverConfiguration.isDeleteTmpFiles()) {
+              androidAPK.deleteOnExit(); //Deletes temporary files if flag set
```

Comment:
actualy, i take that back, the config should stay... but we should default it to true and let people disable this as the exception to the rule :)